lib/cluster-store.js

```diff
@@ -59,6 +59,7 @@ module.exports = function(connectOrSession) {
   function ClusterStore(options) {
     Store.call(this, options);
     this._collection = NativeStore.collection(COLLECTION_NAME);
+    this._collection.configure(options);
```

Comment:
> In documentations says that the options object is to initialise ClusterStore

Well yes, we are passing these options to `session.Store` constructor.

> So, I would say it's better to move collection config to options.config because method config should be called after setting collection name.

Sounds good. I am proposing a small change - let's call the property `options.collection` or `options.collectionConfig`, to make the purpose more clear.